examples/asr/asr_chunked_inference/ctc/speech_to_text_buffered_infer_ctc.py

```diff
@@ -161,6 +183,32 @@ def autocast():
         )
         return cfg
 
+    # Setup decoding strategy
+    if hasattr(asr_model, 'change_decoding_strategy'):
+        if not isinstance(asr_model, EncDecCTCModel) and not isinstance(asr_model, EncDecHybridRNNTCTCModel):
+            raise ValueError(
+                "The script supports ctc model and hybrid model with ctc decodng! use rnnt/speech_to_text_buffered_infer_rnnt.py for other conditions."
+            )
+
+        else:
+            if cfg.compute_langs:
+                raise ValueError("CTC models do not support `compute_langs` at the moment.")
+
+            # ctc model
+            if isinstance(asr_model, EncDecCTCModel):
+                asr_model.change_decoding_strategy(cfg.ctc_decoding)
+
+            # hybrid ctc rnnt model with decoder_type=ctc
+            if isinstance(asr_model, EncDecHybridRNNTCTCModel):
+                if cfg.decoder_type != "ctc":
```

Comment:
change_decoding_strategy should be always called with decoder_type=ctc here as this script is just for ctc. If we drop the decoder_type, it makes it easier to handle the cases.